node/resolvers/catalog/itemMetadata.ts

```diff
@@ -55,9 +55,10 @@ const fetchPrice = async ({
     ...(isEmpty(marketingData) ? {} : { marketingData }),
   }
   const orderForm = await http.post(url, payload, { headers }).catch(() => null)
+  const shouldCheckTotals = orderForm && path(['data', 'totals', 'length'], orderForm)
```

Comment:
I'm not totally familiar with ramda, but is it necessary that you check if `orderForm` exist before you use `path`? I would expect `path` could deal with `orderForm` beeing `null`.

Reply:
Nardi is right

Reply:
@felippenardi `ramda`'s typings don't accept `null` in its `path` function, and `eslint` would complain about that since our HTTP calls returns something like `FetchResponse | null`. It's not possible to link nor publish without this check, I've commented about it in the PR description.

Reply:
![image](https://user-images.githubusercontent.com/18706156/55505134-d1cc0480-5628-11e9-845a-5444e52d1d11.png)